admin-dev/themes/new-theme/js/app/pages/translations/components/app.vue

```diff
@@ -59,24 +59,26 @@
 </template>
 
 <script lang="ts">
-  import Vue from 'vue';
+  import {defineComponent} from 'vue';
   import Search from '@app/pages/translations/components/header/search.vue';
   import Sidebar from '@app/pages/translations/components/sidebar/index.vue';
   import Principal from '@app/pages/translations/components/principal/index.vue';
+  import TranslationMixin from '@app/pages/translations/mixins/translate';
   import PSModal from '@app/widgets/ps-modal.vue';
 
-  export default Vue.extend({
+  export default defineComponent({
     name: 'App',
+    mixins: [TranslationMixin],
     computed: {
       isReady(): boolean {
         return this.$store.getters.isReady;
       },
       totalTranslations(): string {
         return this.$store.state.totalTranslations <= 1
           ? this.trans('label_total_domain_singular')
-            .replace('%nb_translation%', this.$store.state.totalTranslations)
+            .replace('%nb_translation%', this.$store.state.totalTranslations.toString())
```

Comment:
Plutot que d'appeler toString() on pourrait avoir un `get() {
    this.$store.state.totalTranslations.toString()
  },` ?

Reply:
On the component scope? Looks like we can't, we could on store scope but we definitely don't want to have a string everywhere in the app, this is a specific need :/